filebeat/modules.d/nginx.yml.disabled

```diff
@@ -0,0 +1,16 @@
+#- module: nginx
```

Comment:
I think all these should be un-commented? Otherwise they are not really enabled.